desktop-widgets/tab-widgets/TabDiveEquipment.cpp

```diff
@@ -101,18 +135,22 @@ void TabDiveEquipment::updateData()
 {
 	cylindersModel->updateDive();
 	weightModel->updateDive();
+	suitModel.updateModel();
 
 	ui.cylinders->view()->hideColumn(CylindersModel::DEPTH);
 	if (get_dive_dc(&displayed_dive, dc_number)->divemode == CCR)
 		ui.cylinders->view()->showColumn(CylindersModel::USE);
 	else
 		ui.cylinders->view()->hideColumn(CylindersModel::USE);
+	if (current_dive && current_dive->suit)
+		ui.suit->setText(QString(current_dive->suit));
```

Comment:
If current_dive is not set, then this should probably clear the suit field.

Apart from that - looks good to me.

I will test it tomorrow.

Reply:
done